tests/ForeignKeyViolationError.js

```diff
@@ -50,6 +50,11 @@ module.exports = (session) => {
               expect(error.table).to.equal('source');
               expect(error.constraint).to.equal('source_foreign_key_foreign');
             }
+
+            if (session.isMssql()) {
+              expect(error.table).to.equal('dbo.target');
```

Comment:
This reports different table than on `postgres` and `mysql`. I'm not sure if we should use the same property for it. Also why is the `dbo` prefix there?

Reply:
Oh, `dbo` is the schema right? If you want to add that information too, could we add a separate `schema` property?

Reply:
I left some notes in my first comment. So all the errors have at-least the schema. and one (for whatever reason) includes the database too. 

So I kind of defer to you on how to handle this.

Happy to remove the database from that one specific error and then ensure schema is always split out? Which would mean adding a top level schema field to DBError?

Reply:
I didn't add the extra properties initially because it would be null for every single other client

Reply:
I think it's still better than having the same property that cannot be used the same way.

Reply:
So to sum up, I think `table` should only hold the table name. If you think `schema` and `database` are useful, add separate properties for them. If not, just ignore the information.

The bigger problem with this particular error is that it points to the `source` table instead of `target` on other databases. Is there a way to dig out the other table from the error message?

Reply:
Sounds good to me, will try to save those values.

No unfortunately not, the error only contains the one table.